NKL/Trace.lean

```diff
@@ -3,7 +3,18 @@ Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Paul Govereau
 -/
+import NKL.KLR
+import NKL.Python
 import NKL.Trace.Types
 import NKL.Trace.Basic
 import NKL.Trace.Builtin
---import NKL.Trace.Python
+import NKL.Trace.Python
+import NKL.Trace.NKI
+
+namespace NKL.Trace
+
+def runNKIKernel (k : NKL.Python.Kernel) : Except String (List NKL.KLR.Stmt) :=
+  tracer ⟨ .ofList nki_env, #[] ⟩ do
+    traceKernel k
```

Comment:
This is a kind of annoying aspect of Lean imo. Using the namespace mechanism, I now have no idea where to look for `traceKernel` without UI support. I wonder if we just get used to it, or have some conventions around files vs namespace organization.

Reply:
This is my gut feeling, but I may be missing some bigger picture here. I just feel like all module systems I'm aware of that don't respect the file system are harder to use/find things.